### PR TITLE
fix(70309): fix disable button apply on cancel table

### PR DESCRIPTION
**Pull request by imam-auliya**
> [#70309] - FE POS - Done

Acceptance Criteria
- [x] Disable apply button when cancelation note is empty

Changelog:  
- #70309 handle disabled
image

### CODE REVIEW

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=45) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=45)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=45)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=45)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=45)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=45&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications: **Duplication**

File: src/app/modules/order/page/OrderPage.tsx [hunk 1]
```diff
@@ -713,6 +713,7 @@ const OrderPage: FC = () => {
         handleClear={() => {
           setCancellNote('')
         }}
+        disabledButton={cancellNote === ''}
```
Comment (mshafrizalait) [inline src/app/modules/order/page/OrderPage.tsx:716]: state awalnya undefined mas, menurut saya lebih baik disablednya seperti ini `disabledButton={!cancellNote}` atau disabledButton={cancellNote === '' || cancellNote === undefined}

Comment (imam-auliya) [reply src/app/modules/order/page/OrderPage.tsx:716]: Ok thanks mas

**Review by aldiz1711 — APPROVED** [review]
OK

**Pull request merged**